examples/playwright/src/tests/theia-explorer-view.test.ts

```diff
@@ -14,23 +14,30 @@
 // SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
 // *****************************************************************************
 
-import { expect } from '@playwright/test';
+import { expect, test } from '@playwright/test';
+import { TheiaBrowserAppLoader } from '../theia-app-loader';
 import { TheiaApp } from '../theia-app';
 import { DOT_FILES_FILTER, TheiaExplorerView } from '../theia-explorer-view';
 import { TheiaWorkspace } from '../theia-workspace';
-import test, { page } from './fixtures/theia-fixture';
-
-let app: TheiaApp;
-let explorer: TheiaExplorerView;
 
+// the tests in this file reuse a page to run faster and thus are executed serially
+test.describe.configure({ mode: 'serial' });
```

Comment:
Is this fixing a problem you found or is it related to electron? Also: is this a per-file setting or what's the scope?

Reply:
That is required for the stability of the testsuite, which is improved if the tests are run sequentially. When the tests are run in parallel, we experience a LOT more flakiness. Scope is per file to my knowledge.